src/main/java/org/opentripplanner/index/model/RealtimeVehiclePosition.java

```diff
@@ -0,0 +1,27 @@
+package org.opentripplanner.index.model;
+
+import com.google.transit.realtime.GtfsRealtime.VehiclePosition.CongestionLevel;
+import com.google.transit.realtime.GtfsRealtime.VehiclePosition.VehicleStopStatus;
+
+/**
+ * A simplified vehicle position class that supports being output to HTTP.
+ * Compatible with https://github.com/opentripplanner/otp-ui/blob/master/packages/core-utils/src/types.js#L86
+ */
+public class RealtimeVehiclePosition {
+    public String vehicleId;
+    public String label;
+    public double lat;
+    public double lon;
+    public double speed;
+    public double heading;
+
+    /** Unix timestamp of when the realtime position was recorded */
+    public long seconds;
+
+    public String patternId;
+
+    public VehicleStopStatus stopStatus;
+    public String nextStop;
```

Comment:
I propose renameing the to `nextStopName`. Also, the stop ID and stop sequence ID should also be outputted.

Reply:
I support your proposal! Refactored in 1360e72e44678135324c29f7ac4942514fa05270